syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.tardate.lck.explainer";
option java_outer_classname = "ExplainerProto";
option objc_class_prefix = "EXP";

package explainer;

service ShiFu {
  rpc TellMeWhy (ExplainerRequest) returns (ExplainerReply) {}
}

message ExplainerRequest {
  string problem = 1;
}

message ExplainerReply {
  string explanation = 1;
}
